src/modules/tasks.js

```diff
@@ -0,0 +1,105 @@
+const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+function saveTasks() {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+export function renderTasks() {
```

Comment:
- [ ] Kindly make sure to break your code down, you should have three different functions for adding, editing and deleting tasks from your app.